roles/keystone/handlers/main.yml

```diff
@@ -1,16 +1,4 @@
 ---
-- name: pip install keystone
-  action: command pip install -i {{ openstack.pypi_mirror }} /opt/stack/keystone
-  register: result
-  until: result|success
-  retries: 3
-  delay: 60
-  notify: update CA certs
-
 - name: restart keystone services
   restart_if_present: service=keystone
   when: restart|default('True')
-
-- name: restart logstash-forwarder
-  service: name=logstash-forwarder state=restarted
```

Comment:
Could you mention in the commit message that this handler is removed as nothing references it? Might save confusion down the road as to why it got removed in this commit.